roles/common/restart_server/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Restart server # create role for reboot
+  become: yes
+  shell: sleep 2 && reboot
+  async: 1
+  poll: 0
+
+- name: Wait 30 seconds for server to come back
```

Comment:
The description says 30s, while the action says 20.
Suggest updating the description to remove the number.